lib/routes/api/speech-credentials.js

```diff
@@ -236,6 +237,11 @@ const encryptCredential = (obj) => {
       const elevenlabsData = JSON.stringify({api_key, model_id, options});
       return encrypt(elevenlabsData);
 
+    case 'speechmatics':
+      assert(api_key, 'invalid speechmatics speech credential: api_key is required');
```

Comment:
Sorry @xquanluu I was wrong about something -- in addition to the api key speechmatics also requires the URL of the endpoint to connect to as they provide regional endpoints and different endpoints for enterprise customers, as per [here](https://docs.speechmatics.com/introduction/authentication#supported-endpoints).

as far as the credential object goes, let's store the actual URL (e.g. eu2.rt.speechmatics.com) in a property named 'url'.

Reply:
fixed, added speechmatics_stt_uri to speech credential